src/ucs/memory/memtype_cache.c

```diff
@@ -176,11 +182,13 @@ UCS_PROFILE_FUNC_VOID(ucs_memtype_cache_update_internal,
             goto out_unlock;
         }
 
-        ucs_trace("memtype_cache: removed " UCS_PGT_REGION_FMT " %s dev %s",
+        ucs_trace("memtype_cache: removed " UCS_PGT_REGION_FMT " %s dev %s"
+                  " base_addr %p alloc_length %ld",
                   UCS_PGT_REGION_ARG(&region->super),
                   ucs_memory_type_names[region->mem_info.type],
                   ucs_topo_sys_device_bdf_name(region->mem_info.sys_dev,
-                                               dev_name, sizeof(dev_name)));
+                                               dev_name, sizeof(dev_name)),
+                  mem_info->base_address, mem_info->alloc_length);
```

Comment:
1. do we need to update base_address / alloc_length in lines 201-214 below?
2. do we need a test for memtype cache base_address/alloc_length functionality?